lib/containers/cli.rb

```diff
@@ -18,10 +20,6 @@ class CLI < Thor
 
     protected
 
-    def project_name
-      File.basename(Dir.pwd).parameterize
-    end
-
```

Comment:
Should we move this logic as a default to the `Configurable` concern? i.e. Having configuration be optional rather than a requirement.

Reply:
Just realized that you already did this. 🤦🏻‍♂️